### ISSUE TITLE
Map newer HTTP Server Semantic Conventions to URL

### ISSUE THREAD

**Issue by stevejgordon** [elastic/apm-data#254]
Since [v1.21.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/trace/semantic_conventions/http.md#http-server-semantic-conventions) of the trace Semantic Conventions, new attributes are used:

- `net.host.name` -> `server.address`
- `net.host.port` -> `server.port`
- `http.target` -> `url.path` and `url.query`
- `http.scheme` -> `url.scheme`

We currently don't map all of these for the OTLP intake, resulting in a poor UI experience. It seems that only the scheme is coming through.

For example, the trace sample UI isn't able to display the URL of the request which started the transaction.

![image](https://github.com/elastic/apm-data/assets/3669103/06fa9595-7aa9-4f9d-8303-e61567b85404)

Should be closer to:

![image](https://github.com/elastic/apm-data/assets/3669103/82395347-81f8-42ae-b4f1-075a7d67d97d)

**Comment by simitt** [comment]
@stevejgordon would you have capacities to take this one on?

**Comment by stevejgordon** [comment]
@simitt Yes, I'd like to give it a go. Sorry, I thought I'd self-assigned when I opened it! I'm hoping to make some time next week to look at this.

**Comment by simitt** [comment]
No rush from my side; thanks for taking it on.
